packages/v4/patternfly-docs/content/accessibility/navigation/navigation.md

```diff
@@ -0,0 +1,77 @@
+---
+id: Navigation
+section: components
+---
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+
+## Accessibility
+
+To implement an accessible PatternFly **navigation** component:
+- Provide an accessible name for the navigation via the `aria-label` attribute
+- For grouped or expandable nav, add a `title` to each section to distinguish them for users. If `title` is not used, an `aria-label` must be used on the `NavGroup` or `NavExpandable` component.
+
+
+
+## Testing
+
+At a minimum, a navigation component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-1" label={<span>The navigation has an accessible name via the <code class="ws-code">aria-label</code></span>} description="This will differentiate each navigation when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="navigation-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between focusable elements within the navigation." description={<span><kbd>Tab</kbd> navigates to the next focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element. Expandable items can be opened with <kbd>Enter</kbd> or <kbd>Space</kbd>. Flyout menu items use arrow keys to navigate between flyout items, and press <kbd>Escape</kbd> or left arrow to close a flyout. When the navigation is collapsed, you should not be able to navigate to any of the elements.</span>} />
```

Comment:
Ok, I added a section for flyout. I didn't for drilldown yet because I noticed that it seems to be broken at the moment. 🙃 Should I just combine the sections or not mention it at all yet or another option?

Reply:
Ah is it the issue where the menu height shrinks and nothing inside it is actually visible? There was a similar issue with the Menu drilldown, I can take a look to see if its the same issue here.

We could try using the Menu Drilldown to base the verbiage in this PR off of, I imagine the keyboard interaction would be essentially similar. Though it could also be an easy update to make after the fact once the Nav Drilldown example is working properly just to ensure the interaction is accurate.